src/platform/user/profile/vet360/containers/Vet360ProfileField.jsx

```diff
@@ -246,13 +249,21 @@ const mapStateToProps = (state, ownProps) => {
   const data = selectVet360Field(state, fieldName);
   const isEmpty = !data;
   const useAddressValidation = vaProfileUseAddressValidation(state);
+  const addressValidationType =
+    state.vet360.addressValidation.addressValidationType;
+  const showValidationModal =
```

Comment:
I made sure to add plenty of tests for this value since there are so many things that need to be right for it to be truthy.